server/coordinator_test.go

```diff
@@ -830,7 +835,7 @@ func (s *testOperatorControllerSuite) TestStoreOverloadedWithReplace(c *C) {
 	cfg, opt, err := newTestScheduleConfig()
 	c.Assert(cfg, NotNil)
 	c.Assert(err, IsNil)
-	cfg.StoreBalanceRate = 10
+	cfg.StoreBalanceRate = 30
```

Comment:
Could you explain why 30 is the proper value here?

Reply:
I mean, add the comment.

Reply:
Done. PTAL @shafreeck